R/upload.R

```diff
@@ -346,11 +345,11 @@ check_duplicated_inds <- function(registry, df, ind, conf, pool) {
     df_duplicated <- df %>%
       dplyr::left_join(ind_id_type, by = "ind_id") %>%
       dplyr::left_join(orgnr, by = "orgnr") %>%
-      dplyr::filter(!type %in% conf$var$fraction$type) %>%
-      dplyr::select(ind_id, orgnr, context, unit_level, year) %>%
+      dplyr::filter(!.data$type %in% conf$var$fraction$type) %>%
+      dplyr::select(.data$ind_id, .data$orgnr, .data$context, .data$unit_level, .data$year) %>%
```

Comment:
Skal vi bruke `.data` i `dplyr::select`? Får advarsler når tester kjøres lokalt, f.eks. 

> Use of .data in tidyselect expressions was deprecated in tidyselect 1.2.0.
ℹ Please use `"orgnr"` instead of `.data$orgnr`

Reply:
takk:) fikset nå